example/src/EthereumEventsButtons.tsx

```diff
@@ -0,0 +1,48 @@
+import { useWeb3React } from '@web3-react/core'
+
+export const EthereumEventsButtons = () => {
+  const { isActive, provider, chainId } = useWeb3React()
+
+  const sendTransaction = async () => {
+    if ([137, 1].includes(chainId!)) {
+      alert('Transaction is not executed. Change to testnet in order not to lose your money')
+      return
+    }
+    const signer = provider?.getSigner()
+    await signer?.sendTransaction({
+      to: '0x0000000000000000000000000000000000000000',
+      value: '10000000000'
+    })
+  }
+
+  const signMessage = async () => {
+    const exampleMessage = 'Example `personal_sign` message'
+    const signer = provider?.getSigner()
+    const sign = await signer?.signMessage(exampleMessage)
+    console.log({ sign })
```

Comment:
👀

Reply:
I left it in case if someone intersted how sign looks like. But I agree, I'll remove it :)